newsroom/email.py

```diff
@@ -226,19 +231,23 @@ def get_language_template_name(template_name: str, language: str, extension: str
 TemplateKwargs = Dict[str, Any]
 
 
-# TODO-ASYNC: change this to use newsroom.users.model.UserResourceModel instead
+# TODO-ASYNC: change this to use newsroom.users.model.UserResourceModel only
```

Comment:
In my current task I am moving ALL resource models to `newsroom.types` module, so we can safely import types without circular dependency hell.